### PR TITLE
Add threat stack integration

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/5866
Do not merge yet as we didn't test it against an instance

### CODE REVIEW

**Comment by liorkol** [comment]
@Spivaka Let me know when it is ready

File: Integrations/integration-Threat_Stack.yml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
...oJpRzIPD1v60wrBkzEaMmTR/OWghwfpSflnzn0IMh7Fj74vbu9uZ0IeFHKUXDkYOI9faclfXlXCF7uvvqt92c1zx/Tj0r4wgGArS4OIpjRw5tkmXl5SMH9t2l6vqtvmDwGkLIdhCCktIyaCcdn08AYIQQzXXd0clEHNU1IzBh8lS8830fxqx5C72g3DYURZFD4fBIz4pkI+NPyHLq16WUtnqDW6lj2zcxxmaUlVfG/3rfXVu6O9odKklIJROJA7t3vBAIhFzXdec6jv1exliwpLQMf77ndy2JRJxruoYxEycPmNwZYygpLTv43BOP3Lhv1/aZBJiJgZ+bAewtVD6SJKG1uSnQ0dq8TdX150+ekCMlkZKS8r/88ffQfX5v1j97H1JFUdHW3NSybtWK71NCjvT19GDRsis/eOO733cLc90B/obMceA4DmRZRltrS9pxbE4lCa7jjDbS6VKfL4CxEybB7w/AtiyAQEYmpttoABfgZEaWYYWEYaS72lqa27zDK6WObc93bJtWjxgJTdNw8SWXwfYOTOWKEzKw7x4H0Og4DnSfv7a0rLLKMgwY6RSmzZ6La256F0aNHY8Ro8ZkBKJljaGUFudM9u4pHhS2ZaE4EsXciy/JaANCcttqQVHIOcOEKRdg1JjxMFJJ6D7fEH+wTGggn9nT3fmD3Tu2znRseyYhZHAb+h4GhsAxvPfmjDGttKxiom07sCwL5RXVGDGqHrZlwR0UOokxhnAkgtHjJww4Gfu69Htg76F9ex7x+fxwHIdQSt86ceq0sbrPf4VlWWFFUe1D+3c/B/Aj3oEU9HR1gVJpqODkONxw5NCNpmHkK5tLvMXIP7X7QiUJ7//E56BpOnq6ulA7sl591wc+WtvR2gya2UY/isFRATgopRKMdGplZ0fbAsZYvue7gVK6vae7Cx2tLQO27r1wNYe7OtvnOo49k1I6+NpPAIgdPbg/71yWjMe+k0zEB19TqM0MmUMURcUrLz2P+3//6x+qmv6CLMsIBEMLbdv6oZFOD+hk8XgMt37kP1E/bgJs28LMixYwx3Hucx3niux3BsNFMx+853czn3v8kUs556sIIRyZ7EsFF4q2aa5gjM0o8A43AGg5uX57/T+MMZSXlQph+EbBXAf9qakKNl72CnfdHada1GeEkrJV0/0bhitQRdWwYvnTOHJwPxRFPW/LjhCCiZO
```
Comment (liorkol) [inline Integrations/integration-Threat_Stack.yml]: Typo

Comment (liorkol) [inline Integrations/integration-Threat_Stack.yml]: Wrong description

File: Integrations/integration-Threat_Stack.yml [hunk 2]
```diff
@@ -0,0 +1,190 @@
+commonfields:
+  id: Threat Stack
+  version: -1
+name: Threat Stack
+display: Threat Stack
+category: Data Enrichment
```
Comment (liorkol) [inline Integrations/integration-Threat_Stack.yml]: Missing "system: true"

File: Integrations/integration-Threat_Stack.yml (hunk 1 above)
Comment (liorkol) [inline Integrations/integration-Threat_Stack.yml]: Typo repeated

Comment (liorkol) [inline Integrations/integration-Threat_Stack.yml]: Rephrase

Comment (liorkol) [inline Integrations/integration-Threat_Stack.yml]: Unnecessary

**Comment by Spivaka** [comment]
@liorkol your remarks look strange and don't refer to specific code / line numbers

**Comment by meirwah** [comment]
@liorkol @Spivaka what's with this branch?

**Comment by meirwah** [comment]
@Spivaka  what's the status of this?

**Pull request closed without merging**